### ISSUE TITLE
Error sorting a date column with empty values

### ISSUE THREAD

**Issue by komhu84** [olifolkerd/tabulator#699]
When I order a column with both date values and empty values, sorting doesn't work well.
I resolve the problem adding this code in **date** function inside **sort.prototype.sorters:**

```
a = moment(a, format);     
b = moment(b, format);
if (!a.isValid()) a = moment("01/01/1970", format); //Added code.
if (!b.isValid()) b = moment("01/01/1970", format); //Added code.
```

Enjoy
#

**Comment by olifolkerd** [comment]
Hey @komhu84,

Thanks for the suggestion, i will include it in the next patch release.

Cheers

Oli

**Comment by olifolkerd** [comment]
Hey @komhu84 

You will be happy to hear i have just pushed a fix for this to the 3.4 branch, all sorters will now handle unexpected values.

Version 3.4 should be released in the next couple of weeks.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Tabulator version 3.4 has now been officially released!

You can find details of the new release in the [News Section](http://tabulator.info/news)

Full documentation of all the new features can be found [Here](http://tabulator.info/docs/3.4)

Cheers

Oli :)
